testsuite/forge-test-runner-template.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     - -c
     - |
       ulimit -n 1048576
-      forge --suite {FORGE_TEST_SUITE} test k8s-swarm --image-tag {IMAGE_TAG} --namespace {FORGE_NAMESPACE} {REUSE_ARGS} {KEEP_ARGS} {ENABLE_HAPROXY_ARGS}
+      forge --suite {FORGE_TEST_SUITE} --duration-secs {FORGE_RUNNER_DURATION_SECS} --avg-tps {FORGE_RUNNER_TPS_THRESHOLD} test k8s-swarm --image-tag {IMAGE_TAG} --namespace {FORGE_NAMESPACE} {REUSE_ARGS} {KEEP_ARGS} {ENABLE_HAPROXY_ARGS}
```

Comment:
You'll also need to template `FORGE_RUNNER_DURATION_SECS` and `FORGE_RUNNER_TPS_THRESHOLD` in via the `sed` command on line 214 of `run_forge.sh`